### PR TITLE
Add a push command

### LINKED ISSUE #7: Implement a `push` command
The `push` command will look in `.cogs/sheet.tsv` and for each row:

1. put the content of the corresponding TSV into the Google Sheet
    - create a new sheet if it does not exist
    - replace the content of the sheet if it does exist
2. copy the file to `.cogs/` for later comparison

### PR DESCRIPTION
**Pull request by beckyjackson**
Resolves #7 

I reworded some things as some of the terminology gets a bit confusing. Let me know if anything in the README doesn't make sense (sheets vs. worksheets vs. tables... it's a lot...)

### `push`

Running `push` will sync the Google Sheet with your local changes. This includes creating new Worksheets for any added tables (`cogs add`) and deleting Worksheets for any removed tables (`cogs rm`). Any changes to the local tables are also pushed to the corresponding Worksheets.

```
cogs push
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ we try to follow the familiar `git` interface and workflow:
 
 - [`cogs init`](#init) creates a `.cogs/` directory to store configuration data and creates a Google Sheet for the project
 - [`cogs share`](#share) shares the Google Sheet with specified users
-- [`cogs add foo.tsv`](#add) starts tracking the `foo.tsv` table
-- `cogs push` pushes local tables to the Google Sheet
+- [`cogs add foo.tsv`](#add) starts tracking the `foo.tsv` table as a Worksheet
+- [`cogs rm foo.tsv`] stops tracking the `foo.tsv` table as a Worksheet
```
Comment (jamesaoverton) [inline README.md]: This link is broken

**Comment by jamesaoverton** [comment]
A quick test worked properly. I noticed one broken link. The code looks good.

Out of all the choices, why did you settle on "sheet"/"worksheet"?

**Comment by beckyjackson** [comment]
That's the terminology that `gspread` and Google use, so I wanted to keep it consistent.

**Comment by jamesaoverton** [comment]
I agree that it should be consistent. From what I see:

1. Google calls them “spreadsheet” and “sheet”: https://developers.google.com/sheets/api/reference/rest

2. Gspread calls them “spreadsheet” and “worksheet”: https://gspread.readthedocs.io/en/latest/user-guide.html#selecting-a-worksheet

I haven’t seen any examples of “sheet” and “worksheet”. Can you point to any?

I guess I prefer 1. See also https://github.com/ontodev/cogs/issues/2#issuecomment-645521477

**Comment by beckyjackson** [comment]
Oh, I guess I was conflating the two. I'm happy with 1, I can make the
changes in the docs.

**Pull request merged**